models/stg_qualtrics.yml

```diff
@@ -284,8 +284,8 @@ models:
             - version_id
             - survey_id
     columns:
-      - name: _fivetran_deleted
-        description: "{{ doc('_fivetran_deleted') }}"
+      - name: _fivetran_deleted 
+        description: "{{ doc('_fivetran_deleted') }}" 
```

Comment:
This field is not in the final model. We should remove it from the staging docs if that is the case.